single-line-diagram/single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/TestFeederInfos.java

```diff
@@ -106,26 +122,52 @@ void testAllPossibleInfoItems() {
         layoutParameters.setSpaceForFeederInfos(100);
         svgParameters.setFeederInfosIntraMargin(5)
                 .setPowerValuePrecision(0)
-                .setDisplayCurrentFeederInfo(true);
+                .setCurrentValuePrecision(0)
+                .setPercentageValuePrecision(1);
+        DefaultLabelProvider labelProvider = getDefaultDiagramLabelProvider();
+        labelProvider.setDisplayCurrentFeederInfo(true);
+        labelProvider.setDisplayArrowForCurrentFeederInfo(true);
+        labelProvider.setDisplayPermanentLimitPercentageFeederInfo(true);
+
         // build graph
-        network.getLoad("l").getTerminal().setP(100).setQ(10).getBusView().getBus().setV(vl.getNominalV());
+        network.getLoad("l").getTerminal().setP(100).setQ(10);
+        network.getLine("line").getTerminal1().setP(100).setQ(10).connect();
+        network.getLine("line").getTerminal2().setP(90).setQ(10).connect();
+
+        network.getLine("line").getTerminal1().getBusView().getBus().setV(vl.getNominalV());
+        network.getLine("line").getTerminal2().getBusView().getBus().setV(vl2.getNominalV());
```

Comment:
After this modification, convert vl2 field to a local variable
```suggestion
        network.getLine("line").getTerminal1().getBusView().getBus().setV(380);
        network.getLine("line").getTerminal2().getBusView().getBus().setV(380);
```